File: app/src/main/java/com/grable/overflowingstack/models/Question.java
package com.grable.overflowingstack.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by timgrable on 10/31/17.
 */

public class Question {

    @SerializedName("question_id")
    @Expose
    private long question_id;

    @SerializedName("title")
    @Expose
    private String title;

    @SerializedName("body")
    @Expose
    private String body;

    @SerializedName("body_markdown")
    @Expose
    private String body_markdown;

    @SerializedName("creation_date")
    @Expose
    private String creation_date;

    @SerializedName("isGuessed")
    @Expose
    private Boolean isGuessed;

    public long getQuestion_id() { return this.question_id; }

    public void setQuestion_id(long question_id) {
        this.question_id = question_id;
    }

    public String getTitle() {
        return this.title;
    }

    public void setTitle(String postTitle) {
        title = postTitle;
    }

    public String getBody() {
        return this.body;
    }

    public void setBody(String body) {
        this.body = body;
    }

    public String getBody_markdown() {
        return this.body_markdown;
    }

    public void setBody_markdown(String body_markdown) {
        this.body_markdown = body_markdown;
    }

    public String getCreation_date() {
        return this.creation_date;
    }

    public void setCreation_date(String creation_date) {
        this.creation_date = creation_date;
    }

    private Boolean getIsGuessed() {
        return this.isGuessed;
    }
}

File: app/src/main/java/com/grable/overflowingstack/vtwo/models/Question.kt
package com.grable.overflowingstack.vtwo.models

import android.view.View
import com.mikepenz.fastadapter.items.AbstractItem
import android.support.v7.widget.RecyclerView
import com.grable.overflowingstack.R
import kotlinx.android.synthetic.main.question_title.view.*
import com.grable.overflowingstack.vone.models.Answer

/**
 * Created by timgrable on 2/13/18.
 */

class Question(private var question_id: Long = 0,
               private var title: String? = null,
               private var body: String? = null,
               private var body_markdown: String? = null,
               private var creation_date: String? = null,
               private var isGuessed: Boolean? = null,
               private var answers: List<Answer>? = null) : AbstractItem<Question, Question.ViewHolder>() {

    override fun getType(): Int = R.id.response_holder_id
    override fun getViewHolder(v: View): ViewHolder = ViewHolder(v)
    override fun getLayoutRes(): Int = R.layout.question_title

    override fun bindView(holder: ViewHolder, payloads: List<Any>?) {
        super.bindView(holder, payloads)

        holder.itemView.post_title.text = title
        holder.itemView.post_body.text = body
    }

    override fun unbindView(holder: ViewHolder?) {
        super.unbindView(holder)

        holder?.itemView?.post_title?.text = null
        holder?.itemView?.post_body?.text = null
    }

    class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView)
}